doc/api/async_hooks.md

```diff
@@ -4,6 +4,10 @@
 
 > Stability: 1 - Experimental
 
+> We plan to remove this API in the future; it will never become stable.
```

Comment:
```suggestion
> Stability: 1 - Experimental. We plan to remove this API
> in the future; it will never become stable.
```

<details><summary>Before/After</summary>

| Before | After |
| - | - |
| <img width="970" alt="screenshot of the end result as two separate blocks" src="https://user-images.githubusercontent.com/14309773/200172004-0b6509bc-9ddf-4d4e-b8de-0b67a57b7235.png"> | ![screenshot of the end result as a single block](https://user-images.githubusercontent.com/14309773/200171974-350d4129-4a19-488a-ba0b-431d6407b724.png) |

</details>

Reply:
Sure, is this suggested formatting acceptable? It won't break any automation? @Trott? I was copying how we had the loaders "this API will be redesigned" note.

Reply:
No it won't break any automation, and would be consistent with the `fetch` warning:

https://github.com/nodejs/node/blob/396e8e3001f14f1ad62b2ce0aea6ead7adc55719/doc/api/globals.md#L472-L473